File: mbtsdk/src/main/java/core/bluetooth/MbtDataBluetooth.java
package core.bluetooth;


import android.content.Context;

public abstract class MbtDataBluetooth extends MbtBluetooth implements BluetoothInterfaces.IDataBluetooth {

    public MbtDataBluetooth(Context context, BluetoothProtocol protocol, MbtBluetoothManager mbtBluetoothManager) {
        super(context, protocol, mbtBluetoothManager);
    }
}

File: mbtsdk/src/main/java/core/bluetooth/MbtDataBluetooth.kt
package core.bluetooth

import core.bluetooth.BluetoothInterfaces.IDataBluetooth

class MbtDataBluetooth {
companion object{
  lateinit var instance : MainBluetooth
  fun isInitialized() : Boolean{
    return ::instance.isInitialized
  }
}


abstract class MainBluetooth(
    protocol: BluetoothProtocol,
    manager: MbtBluetoothManager) :

    MbtBluetooth(
    protocol,
    manager),

    IDataBluetooth

}